regress/core/in_marc21.sql

```diff
@@ -0,0 +1,856 @@
+--
+-- GeomFromMARC21 regression test
+-- Copyright (C) 2021 University of Münster (WWU), Germany
+-- Written by Jim Jones <jim.jones@uni-muenster.de>
+
+-- BEGIN stress test 
+
+-- Creates random unicode strings with a given length 
+CREATE OR REPLACE FUNCTION public.random_unicode_string(int)
```

Comment:
I feel we do NOT want "random" stuff in testsuite. Is the test for robustness/fuzzing tests ? Because in that case those should probably go to what we call "garden test".

Reply:
This random test was an attempt to break the function with unexpected input characters. Now that you mention it, I also believe it might better fit in your garden test. I will remove it from the test files.

Reply:
Tests with random strings removed #[e098958](https://github.com/postgis/postgis/pull/659/commits/e0989587dacd89e140766fcde2ec6de73c3c1c61)